assets/prometheus-adapter/service-monitor.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    name: prometheus-adapter
+  name: prometheus-adapter
+  namespace: openshift-monitoring
+spec:
+  endpoints:
+  - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+    interval: 30s
+    port: https
+    scheme: https
+    tlsConfig:
+      insecureSkipVerify: true
```

Comment:
Should I add the certs in a follow-up PR?

Reply:
We have to do it here :-) look for a reference i.e. here: https://github.com/openshift/cluster-monitoring-operator/blob/4b907eac67450edad388fe04f51343f8d6582d6d/jsonnet/prometheus-operator.jsonnet#L104-L119